Core/Git/AncestorFinder.cs

```diff
@@ -63,7 +63,8 @@ public string GetAncestor (params string[] expectedAncestors)
     }
 
     _log.Warning("Multiple matching ancestors were found: '{FoundAncestors}'", foundAncestors);
-    var input = _inputReader.ReadStringChoice("Please enter the name of the ancestor branch:", foundAncestors);
+    var input = _inputReader.ReadStringChoice(
+        "Found multiple branches that could be the ancestor branch.\nPlease choose the correct ancestor branch (used to determine which versions are available and which branch to merge into):", foundAncestors);
```

Comment:
Please check, Resharper will format this with a linebreak between the parameters since it will wrap statements that already contain a linebreak at each parameter.

Reply:
Seems my settings aren't correct yet, it did in fact not do that :(